### PR TITLE
Implant Tuneup

### PR DESCRIPTION
**Pull request by ghost**
Another old PR I never bothered finishing. But now it's done. Adds a (currently unused) implant vendor object. Feel free to add it to the maps if you want. Changes implanter sprite to be less shitty, I'm not too proud of it but I think the color-changing effect based on what implant you have is cool. Also, anything's better than the old implanter sprite. 
Also does a lot of codework to change the variable names to be way less shitty plus a decent amount of other minor changes I probably forgot.
:cl:
 * rscadd: New implanter sprite that changes color.

### CODE REVIEW

File: code/modules/medical/cloning.dm [hunk 1]
```diff
@@ -154,22 +154,7 @@
...
-	if (!implanted)
-		return "ERROR"
-	else
-		if(isliving(implanted))
-			var/mob/living/L = implanted
-			healthstring = "[round(L.getOxyLoss())] - [round(L.getFireLoss())] - [round(L.getToxLoss())] - [round(L.getBruteLoss())]"
-		if (!healthstring)
-			healthstring = "ERROR"
-		return healthstring
+//Clonepod
```
Comment (bathosbathtime) [inline code/modules/medical/cloning.dm:157]: Does this PR completely remove the health implant functions of the cloner?

**Comment by SonixApache** [comment]
new sprite is trash tbh

**Comment by ghost** [comment]
I know, but again it's better than 2003 art. Feel feee to try your hand at it. Spr*ters...

File: code/modules/medical/cloning.dm (hunk 1 above)
Comment (ghost) [reply code/modules/medical/cloning.dm:157]: Health implant code was moved to implant.dm. The cloner has nothing to do with it, so it was in a poor spot.

**Comment by SonixApache** [comment]
I like the old one better.

**Comment by bathosbathtime** [comment]
I also like the old one better as the plunger injector is a clear visual as to the fact that it can, and when it can, take an injector from a dish and when it can inject.  The new gun model isn't immediately obvious that it can take implants directly from the dish, or how you would be taking the implant out of the dish.
The new visual for being ready to inject looks good, however, and the sheer novelty of something new might make it worth trying out.

**Comment by ghost** [comment]
Maybe I could try and improve the sprite to be more plunger-y, but I don't know where to go with that.

**Comment by Kurfursten** [comment]
fix conflicts

**Pull request closed without merging**